src/PProf.jl

```diff
@@ -140,8 +140,8 @@ function pprof(data::Union{Nothing, Vector{UInt}} = nothing,
     samples = Vector{Sample}()
 
     sample_type = [
-        ValueType!("events",      "count"), # Mandatory
-        ValueType!("stack_depth", "count")
+        ValueType!("events", "count"), # Mandatory
+        ValueType!("cpu", "nanoseconds")
```

Comment:
can we still emit stack-depth values, or does even their presence break the datadog viewer?

I'm fine to drop them though; i don't think they really added much, and they're just there because Valentin and i were trying to understand how the sample_types worked, i think

Reply:
I'm not sure if they break the DD viewer… I'm not sure why you'd need them though, since each sample itself is a full stack, so the depth information is already there